docassemble/ALToolbox/misc.py

```diff
@@ -174,3 +172,10 @@ def add_records(obj, labels):
         obj[index].reference = val[1]
         index += 1
     return obj
+
+def output_checkbox(value:bool=None):
+    """Generate a conditional checkbox for docx templates"""
+    if value:
+      return '[X]'
+    else:
+      return '[  ]'  
```

Comment:
```suggestion
def output_checkbox(value_to_check:bool, checked_value:str="[X]", unchecked_value:str="[  ]"):
    """Generate a conditional checkbox for docx templates"""
    if value_to_check:
        return checked_value
    else:
        return unchecked_value
```

we shouldn't use the keyword `value` because it's already a Docassemble function.

Reply:
Right!
I'll add an example in docx:

````
Or you can override the default checkbox symbols (strings):

Is there fresh air? {{output_checkbox(fresh_air, checked_value=”(x)”, unchecked_value=”( )”) }}`
```